frontend/src/utils/TaskUtils.ts

```diff
@@ -150,7 +150,7 @@ export const ratedByCustomer = (
       rating.forCustomer === customer.id && rating.createdByUser === rep.id,
   );
 
-const completed = (task: Task) => task.status === TaskStatus.COMPLETED;
+export const completed = (task: Task) => task.status === TaskStatus.COMPLETED;
```

Comment:
Now that this is exported, there would be couple of places to use it. I changed those to use this.

Also, this was named in the single file context, not considering exporting.
I'm not sure if it's the best name (as exported), but I think it's good enough together with the function signature.